packages/ffe-searchable-dropdown-react/src/SearchableDropdown.js

```diff
@@ -264,8 +274,15 @@ SearchableDropdown.propTypes = {
     inputId: isRequiredIf(string, props => props.label),
     /** Value of input field */
     inputValue: string,
-    /** value to be displayed in dropdown in case of search with no match */
-    noMatch: string.isRequired,
+    noMatch: oneOfType([
+        /** value to be displayed in dropdown in case of search with no match */
+        string.isRequired,
+        /** or value to be display along another list of items */
+        shape({
+            text: string.isRequired,
+            items: [],
```

Comment:
` items: array, ` eller ` items: arrayOf`?

Reply:
You're right, this one slipped my mind. I suppose we would want array, since we can't necessarily know the structure of the data?